src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -58,4 +95,68 @@ public String getDescription() {
     public Duration getEstimatedEffortPerOccurrence() {
         return Duration.ofMinutes(5);
     }
+
+    static class RandomCronExpression {
+        private final Random random;
+
+        public RandomCronExpression(Random random) {
+            this.random = random;
+        }
+
+        public int random(int min, int max) {
+            return random.nextInt(max + 1 - min) + min;
+        }
+
+        public int minute() {
+            return random(0, 59);
+        }
+
+        public int hour() {
+            return random(0, 23);
+        }
+
+        public String dayOfWeek() {
+            final List<String> daysOfWeek = Arrays.asList("mon", "tue", "wed", "thu", "fri", "sat", "sun");
+            return daysOfWeek.get(random.nextInt(daysOfWeek.size()));
+        }
+
+        public String dailyCron() {
+            return String.format("%d %d * * *", minute(), hour());
+        }
+
+        public String weeklyCron() {
+            return String.format("%d %d * * %s", minute(), hour(), dayOfWeek());
+        }
+
+        public String weekdays() {
+            return String.format("%d %d * * 1-5", minute(), hour());
+        }
+
+        public String weekends() {
+            return String.format("%d %d * * sat,sun", minute(), hour());
+        }
+
+        public String monthlyCron() {
+
+            return String.format("%d %d %s * *", minute(), hour(), dayOfTheMonth());
```

Comment:
Since `dayOfTheMonth()` returns an `int` shouldn't this be similar to `yearlyCron()` below?
```suggestion
            return String.format("%d %d %d * *", minute(), hour(), dayOfTheMonth());
```